### PR TITLE
1040 update documentation

### PR DESCRIPTION
**Pull request by Orta21**
Ticket: #1040

### Dependencies

- Upstream: none
- Downstream: none

### Description

- Update docs - sdks, postman, openapi and remove org info

### Release Plan

- [ ] Add github secrets
- [ ] Fern Definition Updated
- [ ] Release Fern SDKs
- [ ] Merge this

### CODE REVIEW

File: docs/medical-api/more-info/api-tools.mdx [hunk 1]
```diff
@@ -0,0 +1,137 @@
+---
+title: "Api Tools"
```
Comment (Goncharo) [inline docs/medical-api/more-info/api-tools.mdx:2]: thinking about this some more, should we have 3 separate pages for:

- SDKs
- OpenAPI spec
- Postman collection

"Api Tools" is very vague, and hard to know we even have these available at the surface

File: docs/medical-api/more-info/api-tools.mdx [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+[Metriport OpenAPI Specification](https://github.com/metriport/metriport/tree/develop/fern/definition)
+
+## SDKs
+
+We provide Software Development Kits (SDKs) in multiple programming languages to make it easier for developers to integrate with our API. Our SDKs are available for the following languages:
+
+### Python
+
+- Package: [metriport](https://pypi.org/project/metriport/)
+- Installation: `pip install metriport`
```
Comment (Goncharo) [inline docs/medical-api/more-info/api-tools.mdx:19]: if we're going to add these to docs, we need to maintain them, updated the spec, and re-generate + publish

unless we do that, I'd suggest we only advertise our ts SDK in the docs, and tell people if they want to get SDks in other langs, to generate them off of the OpenAPI spec

File: docs/medical-api/more-info/user-roles.mdx [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+- Can view invoices and manage payment methods
+- No access to clinical data, user management, or any other pages
+
+### Developer
+
+- Can create, read, and update patients
+- Can create, read, and update facilities
+- Can view patients' medical history
+- Can access the developers tab, including API credentials and webhook settings
+- No access to user management or billing pages
```
Comment (Goncharo) [inline docs/medical-api/more-info/user-roles.mdx:69]: would a table be easier to understand? ie what we did before with DAPI to denote which data types which data sources supported

File: docs/medical-api/more-info/api-tools.mdx (hunk 2 above)
Comment (Orta21) [reply docs/medical-api/more-info/api-tools.mdx:19]: Gotcha we currently already do that so ill just leave as is then

File: docs/medical-api/getting-started/quickstart.mdx [hunk 4]
```diff
@@ -12,7 +12,6 @@ Getting started with our Medical API is pretty easy, you'll just need take the f
 1. [Create a developer account](#1-create-a-developer-account).
 1. [Generate an API key](#2-generate-an-api-key).
 1. [Integrate your app with Metriport](#3-integrate-your-app-with-metriport).
-1. [Register your Organization](#4-register-your-organization).
 1. [Create Facilities and Patients](#5-create-facilities-and-patients).
```
[the remark this answers is not in the document]
Comment (Orta21) [reply docs/medical-api/getting-started/quickstart.mdx:15]: Good catch didnt see those on the URL

**Review by Goncharo — APPROVED** [review]
:shipit:

**Pull request merged**